reviewrot/basereview.py

```diff
@@ -123,6 +123,11 @@ def _call_api(self, url, method='GET', ssl_verify=True, ignore_err=False):
         try:
             response = self.get_response(method, url, ssl_verify)
             decoded_response = response.json()
+            # Some services (like pagure) return valid JSON with a 404 error.
+            # https://pagure.io/api/0/username/reponame/pull-requests
+            if not bool(response):
```

Comment:
I suspect you don't need to convert to a bool here. Python should do that automatically.